### PR TITLE
v10

### PR DESCRIPTION
**Pull request by WilliamDenniss**
Addressing changes from last call.

### CODE REVIEW

File: draft-ietf-oauth-device-flow.xml [hunk 1]
```diff
@@ -921,8 +933,8 @@
       <t>
         Roshni Chandrashekhar, Marius Scurtescu, Breno de Medeiros,
         Stein Myrseth, Simon Moffatt, Brian Campbell, James Manger,
-        Justin Richer, Ken Wang, Steven E.&nbsp;Wright, Nat Sakimura, and
-        Torsten Lodderstedt.
+        Justin Richer, Ken Wang, Steven E.&nbsp;Wright, Nat Sakimura,
+        Torsten Lodderstedt, Eric Fazendin, and Andrew Sciberras.
```
Comment (selfissued) [inline draft-ietf-oauth-device-flow.xml]: Please alphabetize this list.

**Review by selfissued — APPROVED** [review]
These changes look good.  I'd alphabetize the acknowledgements by last name before merging, as is standard to do.  Typically I put one name per line so as to minimize changes when more names are inserted into the list.  We'll be adding more during IETF and IESG review.

File: draft-ietf-oauth-device-flow.xml (hunk 1 above)
Comment (WilliamDenniss) [reply draft-ietf-oauth-device-flow.xml]: Done. I also added a credit for our chairs and ADs, and slightly clarified the credit for the earlier contributions.

**Review by selfissued — APPROVED** [review]
Good to go!

**Pull request merged**